packages/mask/src/social-network-adaptor/twitter.com/utils/selector.ts

```diff
@@ -219,7 +219,7 @@ export const searchAvatarSelectorImage = () =>
         .closest<HTMLDivElement>(2)
         .querySelector<HTMLDivElement>('div > div > :nth-child(2) > div > img')
 
-export const searchAvatarOpenFileSelector = () => querySelectorAll<E>('[data-testid="fileInput"]').at(1)
+export const searchTwitterAvatarOpenFilesSelector = () => querySelectorAll<E>('[data-testid="fileInput"]').at(1)
```

Comment:
It's already under a `twitter.com` namespace.

Reply:
Yes, but the naming distinction will prevent the IDE from automatically importing the wrong